Style-Guide.md

```diff
@@ -0,0 +1,53 @@
+# Planet Style Guide
+Below is a brief summary of code standards we'd like to keep with this project.  For the most part this follows the [Angular Style Guide](https://angular.io/guide/styleguide), but in a more concise way so you have a quick to read reference guide.
+## Editor Config
+If you have an IDE that supports a `.editorconfig` file, please make sure you take the necessary steps to install those files.  There's more info on specific IDE installations [here](http://www.editorconfig.org).
+
+This will help with a few things: maintaining 2 space indentations, ensuring there is a newline at the end of each file, and removing unnecessary whitespace at the end of a line.
+## Angular & TypeScript
+### Template & Style
+If the HTML & CSS are fewer than 12 lines total, please keep the `template` and `style` in the `.component.ts` file.  Otherwise, you should move one or both to separate `.component.html` or `.component.scss` file(s).
+
+If there are more than 12 lines, but the component has little to no TypeScript functionality, then an exception can be made if code reviewers agree to it.
+
+### TSLint
+We have a git hook to run TSLint from the Vagrant before pushing code to the repository.  Please install this hook to your local machine by running the following command:
+```
+npm run install-hooks
+```
+The only limitation with TSLint currently is that it is not very good at indentation rules.  Please make sure to comply with 2 space indents and to indent the appropriate number of times.
+### Naming
+We follow the standard Angular naming and would like to stick to simple names for files and classes.  Each name should follow the format:
+```
+<feature><sub-feature?><type>
+```
+For files the different parts should be separated with a dash, for class names we should use camel case.  Here are some examples:
+#### Courses Component
+```
+File: courses.component.ts
+Class name: CoursesComponent
+```
+#### Resources Review Service
+```
+File: resources-review.service.ts
+Class name: ResourcesReviewService
+```
+We suggest using max one sub-feature word to keep file and class names from getting too long.
+### App Directory Structure
+Within the `src/app` directory, each feature should have its own directory.  Within that directory if, as we add sub-features, there are more than 9 files we should create a sub-directory with the same naming convention as files (i.e. Resources Review would be in a `resources-review` directory).
```

Comment:
there should be an `if` after the second comma in the second sentence. `if there`